clusterloader2/testing/network/1_1_ratio_override.yaml

```diff
@@ -0,0 +1,2 @@
+Ratio: "1:1"
```

Comment:
BTW - I think we moved from ratio and now have number of servers/clients, right?

Reply:
yes modified